addons/mail/static/src/components/activity/activity.scss

```diff
@@ -2,68 +2,35 @@
 // Layout
 // ------------------------------------------------------------------
 
-.o_Activity {
-    display: flex;
-    flex: 0 0 auto;
-    padding: map-get($spacers, 2);
-}
-
 .o_Activity_detailsUserAvatar {
-    margin-inline-end: map-get($spacers, 2);
-    object-fit: cover;
-    height: 18px;
-    width: 18px;
+    height: 1.4em;
+    width: 1.4em;
+    vertical-align: text-bottom;
+    margin-inline-end: map-get($spacers, 1);
 }
 
 .o_Activity_dueDateText, .o_Activity_summary {
     margin-inline-end: map-get($spacers, 2);
 }
 
 .o_Activity_iconContainer {
-    @include o-position-absolute($top: auto, $left: auto, $bottom: -5px, $right: -5px);
-    display: flex;
-    align-items: center;
-    justify-content: center;
-    width: 25px;
-    height: 25px;
-    border-width: 2px;
-}
-
-.o_Activity_info {
-    display: flex;
-    align-items: baseline;
+    @include o-position-absolute($top: auto, $left: auto, $bottom: -16%, $right: -5%);
 }
 
 .o_Activity_note p {
     margin-bottom: map-get($spacers, 0);
 }
 
-.o_Activity_sidebar {
-    display: flex;
-    flex: 0 0 36px;
-    margin-right: map-get($spacers, 3);
-    justify-content: center;
-}
-
-.o_Activity_toolButton {
-    padding-top: map-get($spacers, 0);
-}
-
-.o_Activity_tools {
-    display: flex;
+.o_Activity_sidebar, .o_Activity_user {
```

Comment:
Also avoid `,` rules as they get quite hard to follow sometimes.

All the layout for `o_Activity_user` should only be in one place, and same for `o_Activity_sidebar`.

If they need common style, we should either:
- create a new class, to apply on both
- define a mixin and use it on both

Reply:
Ok, I'll reuse the same rule for both selectors. 

> If they need common style, we should either:
>- create a new class, to apply on both elements in the DOM
>- define a mixin and use it on both in css

yep, pseudo-classes are very convenient too :)